src/app/search/search.component.css

```diff
@@ -0,0 +1,102 @@
+header {
+  position: relative;
+  overflow: auto;
+  width: 100%;
+  border-bottom: 1px solid #676767;
+  margin-bottom: 10px;
+}
+header h2 {
+  font-size: 25px;
+  line-height: 30px;
+  color: #343434;
+  margin-bottom: 1px;
+}
+
+input {
+  width: 100%;
+  border: 0;
+  padding: 6px;
+  color: #939393;
+  background: #fff;
+}
+
```

Comment:
Yeah, anything that may eventually end up in the "style guide" show go into one of those global css files for now.